tests/wpt/web-platform-tests/html/semantics/forms/the-input-element/month.html

```diff
@@ -13,19 +13,37 @@
   <body>
     <h1>Inputs Month</h1>
     <div style="display: none">
+      <input id="valid_value_1" type="month" value="20133-12" />
+      <input id="valid_value_2" type="month" value="2013-12" />
+      <input id="valid_value_3" type="month" value="0003-01" />
       <input id="valid" type="month" value="2011-11" min="2011-01" max="2011-12" />
       <input id="invalid_value" type="month" value="invalid-month" min="2011-01" max="2011-12"/>
-      <input id="value_can_be_empty_string" type="month" value="2013-06" />
+      <input id="value_can_be_empty_string" type="month" value="" />
       <input id="invalid_value_with_two_digits_year" type="month" value="13-06" />
       <input id="invalid_value_is_set" type="month" />
       <input id="step_attribute_is_invalid_value" type="month" value="2013-06" step="invalid_step_value" />
       <input id="invalid_month_too_high" type="month" value="2013-13" />
       <input id="invalid_month_too_low" type="month" value="2013-00" />
+      <input id="invalid_year_all_zerro" type="month" value="0000-10" />
+      <input id="invalid_month_with_one_number" type="month" value="2013-1" />
+      <input id="invalid_value_with_four" type="month" value="2013-11-1-1" />
     </div>
 
     <div id="log"></div>
 
     <script>
+      test(function() {
+        assert_equals(document.getElementById("valid_value_1").value, "20133-12")
+      }, "year can be more than four number");
+
+      test(function() {
+        assert_equals(document.getElementById("valid_value_2").value, "2013-12")
+      }, "valid value test");
+
+      test(function() {
+        assert_equals(document.getElementById("valid_value_3").value, "0003-01")
+      }, "year must be four or more number even if it is zero");
```

Comment:
`"year can contain prefixes of zero, as long as there are at least four digits"`